README fixes after Jacek's review (#70)
golemfactory/ray-on-golem: README.md

# Limitations

Current version is `pre-alpha` which means the happy path is working on Ubuntu on the Golem test network.

> We have tested Ray on Golem on Ubuntu and on WSL. It should work on other Linux distributuions, might work on MacOS and shouldn't on bare Windows.

Reviewer: ```suggestion
We have tested Ray on Golem on Ubuntu and on WSL, but it should work on other Linux distributions. At the moment, we don't support MacOS or bare Windows.
```

 
We use this version to show the direction and get feedback.

> There is one Ray on Golem image. It contains `ray 2.3.1` and `python 3.10.12`.

Reviewer: I wouldn't mention patch versions, especially that I believe it's not specifically 3.10 _.12_ right now... 

```suggestion
There is one Ray on Golem image. It contains `ray 2.3` and `python 3.10`.
```
